Codecademy/gamut
# feat(SelectDropdown): updated aria-live to be assertive and added aria-selected attribute to SelectDropdown

### Overview
As part of this PR, the aria-live attribute for SelectDropDown is made assertive. This will help screenreader in identifying changes in option selection done be user and thereby notify without any delay. 

Updated aria-live to be assertive.
Adds aria-selected attribute to option focused/highlighted


### PR Checklist

- [X] Related to JIRA ticket: [ENT-69](https://codecademy.atlassian.net/browse/ENT-69)

#### Testing Instructions

1. Go to SelectDropdown Story
2. Turn on VoiceOver
3. Using TAB open select menu
4.  You should hear different option being read while changing focus on different options under select menu



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/mono/pull/6040)  |  |





[ENT-69]: https://codecademy.atlassian.net/browse/ENT-69?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
